CHANGELOG/CHANGELOG-4.1.md

```diff
@@ -0,0 +1,54 @@
+# Release notes for v4.1.0
```

Comment:
Looks like the README should also be updated

Reply:
The top-level one? Yes, that's severely outdated.

I've pushed a commit which updates it and the mock/README.md.